build-system/tasks/unit.js

```diff
@@ -99,7 +99,7 @@ function getFileConfig() {
     testConfig.chaiAsPromised
   );
```

Comment:
Unit tests can also run a subset of tests using `--files`.

Reply:
Done!